azure-iot-device/azure/iot/device/common/unhandled_exceptions.py

```diff
@@ -0,0 +1,27 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
```

Comment:
Shouldn't this file be in the pipeline subdirectory?

Reply:
I'm working on the assumption that exceptions in background threads will be thrown from other places that are not part of the pipeline.  For example, SAS token renewal takes place based on a timer interval, so this will be on a background thread.  If that fails, we need a place to send that exception.  

---
In reply to: [305478394](https://github.com/Azure/azure-iot-sdk-python-preview/pull/133#discussion_r305478394) [](ancestors = 305478394)